queue.go

```diff
@@ -95,8 +95,16 @@ func (q *Queue) updateTitle() string {
 		count = "song"
 	}
 
-	title := fmt.Sprintf("─ Queue ───┤ %d %s | %s ├",
-		len(q.items), count, fmtTime)
+	var loop string
+
+	if q.isLoop {
+		loop = "ﯩ"
```

Comment:
It'll be great if it respects `general.use_emoji` option

Reply:
Is this good?  2 if embeded seems not easy to read.

```
	if q.isLoop {
		if viper.GetBool("general.emoji") {
			loop = viper.GetString("emoji.loop")
		} else {
			loop = "Loop"
		}
	} else {
		if viper.GetBool("general.emoji") {
			loop = viper.GetString("emoji.noloop")
		} else {
			loop = "No loop"
		}
	}

```

Reply:
It looks good just need a little touch up like this perhaps
```go
	isEmoji := viper.GetBool("general.emoji")

	if q.isLoop {
		if isEmoji {
			loop = viper.GetString("emoji.loop")
		} else {
			loop = "Loop"
		}
	} else {
		if isEmoji {
			loop = viper.GetString("emoji.noloop")
		} else {
			loop = "No loop"
		}
	}
```